test/service/test_response.py

```diff
@@ -2827,10 +2827,16 @@ class TestSchemaTestDataCannedBundle(WebServiceTestCase):
     @classmethod
     def bundles(cls) -> List[BundleFQID]:
         return [
-            # Bundles from the canned staging area, both for project 90bf705c
+            # Bundles for project 90bf705c from commit 7a03924481
```

Comment:
That can't be right. That commit precedes the merging of https://github.com/HumanCellAtlas/schema-test-data/pull/16

Reply:
Now you've gotta somehow prove to me that the cans you committed come from that commit, or from a commit, whatever the right commit is.

Reply:
@hannes-ucsc : "I misinterpreted the history depicted in the screenshot. In reality commit 7a03924481 is the most recent commit on the PR branch, and the subsequent merge should not affect the cans. However, I still think we should re-can the merge commit and have the link in the comment above the test refer to the merge commit as well. Re-canning should not modify the cans."